### PR TITLE
Issue template update

### PR DESCRIPTION
**Pull request by jkrech**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blocker.yaml [hunk 1]
```diff
@@ -46,12 +46,10 @@ body:
       options:
         - label: cbuild
           required: false
-        - label: cbuild-gen
+        - label: csolution
           required: false
         - label: cpackget
           required: false
-        - label: csolution
-          required: false
```
Comment (brondani) [inline .github/ISSUE_TEMPLATE/blocker.yaml:54]: I would recommend to replace `cbuild-gen` by `cbuild2cmake` and keep `csolution` where it is, considering the tools are in alphabetical order.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -40,8 +40,6 @@ body:
       options:
         - label: cbuild
           required: false
-        - label: cbuild-gen
-          required: false
```
Comment (brondani) [inline .github/ISSUE_TEMPLATE/bug.yaml:44]: Same as above.

**Review by jkrech — COMMENTED** [review]
I agree to your changes.

**Review by brondani — APPROVED** [review]
LGTM

**Pull request merged**